engine/crates/engine-v2/src/request/mod.rs

```diff
@@ -37,6 +39,49 @@ pub struct Operation {
 }
 
 impl Operation {
+    fn enforce_operation_limits(&self, schema: &Schema) -> Result<(), OperationLimitExceededError> {
+        // FIXME: Implement.
+
+        let walker = self.walker_with(schema.walker()).walk(self.root_selection_set_id);
```

Comment:
nitpick, would have named it `selection_set` actually

Reply:
Done!